functional-samples/cookbook.offscreen-clipboard-write/manifest.json

```
@@ -2,6 +2,7 @@
   "name": "Offscreen API - Clipboard",
   "version": "1.0",
   "manifest_version": 3,
+  "description": "This recipe shows how to write a string to the system clipboard using the Offscreen API",
```

Could we update the README too?

```suggestion
  "version": "1.0",
  "manifest_version": 3,
  "description": "This recipe shows how to write a string to the system clipboard using an offscreen document",
```